### PR TITLE
Issue #2781: [UX] Menu block settings: Improve the help text of the h…

### PR DESCRIPTION
**Pull request by klonos**
…amburger icon option

Fixes https://github.com/backdrop/backdrop-issues/issues/2781

### CODE REVIEW

**Review by opi — APPROVED** [review]
Approved.
The 2 commits might be squashed on merge to avoid "Update system.menu.inc" message in git history.

File: core/modules/system/system.menu.inc [hunk 1]
```diff
@@ -107,7 +107,7 @@ function system_menu_block_form($config) {
     '#type' => 'checkbox',
     '#title' => t('Display menu toggle button on small screens'),
     '#default_value' => $config['toggle'],
-    '#description' => t('A menu toggle button, commonly known as a hamburger icon, allows the menu to be hidden initially on devices with small screens, but then easily expanded.'),
+    '#description' => t('On devices with small screens, this option reduces the menu to a toggle button, commonly known as <a href="@url" target="_blank"><em>hamburger button</em></a>. Clicking the icon reveals or hides the menu, saving screen space.', array('@url' => 'https://en.wikipedia.org/wiki/Hamburger_button')),
```
Comment (quicksketch) [inline core/modules/system/system.menu.inc]: In this case, the URL should be a part of the string, so the link itself can go to a localized version of the Wikipedia page for Hamburger Icon.

**Review by quicksketch — CHANGES_REQUESTED** [review]

File: core/modules/system/system.menu.inc (hunk 1 above)
Comment (klonos) [reply core/modules/system/system.menu.inc]: Good call @quicksketch 👍  ...done.

**Comment by quicksketch** [comment]
Thanks! Merged into 1.x, will be in 1.8.0.

**Pull request merged**